config/base.json

```diff
@@ -34,7 +34,8 @@
 			"applicationId": "40a2019f-88ad-4076-ab52-ffd074064eed",
 			"clientToken": "pubd5cf774aa1cbbd001accd50cb463925b",
 			"service": "non-prod.developer.hashicorp.com"
-		}
+		},
+		"product_slugs_with_integrations": ["vault", "waypoint"]
```

Comment:
The main difference is that `flags` has been used for just booleans, and this object is used for other data types.

Reply:
Cool cool;  I like your placement then of this object here better.  I do like the language usage of `enabled_*` used in the key on the other PR though!

I will let you & @kendallstrautman decide though what the best key is to manage this, just wanted to share my opinions!

Reply:
If it's alright, it'd be convenient if the rename was handled in the next PR since this isn't my primary focus this week 🙏🏻

Reply:
Yeah that's totally fine!

Reply:
I'm cool with it not being in `flags`, that makes sense. But I do think having this under `dev_dot` is a better place, since integrations don't apply to .io config. But agreed we can adjust that in the refactor PR!

Reply:
It's already under `dev_dot`! 😊

Reply:
Oh I must have misread it! Sorry 😅 all good then